tests/test_cr.py

```diff
@@ -120,8 +121,11 @@ def testDetection(self):
         # Mask known bad pixels
         #
         measAlgorithmsDir = lsst.utils.getPackageDir('meas_algorithms')
-        badPixels = algorithms.Defects.readText(os.path.join(measAlgorithmsDir,
-                                                             "policy", "BadPixels.ecsv"))
+        defectTable = Table.read(os.path.join(measAlgorithmsDir, "policy", "BadPixels.ecsv"))
```

Comment:
This now looks funny because it's divorcing that test file from the `Defects` implementation but we don't really have any use for that file any more.  For the purposes of these tests we'd be better off with a helper function that returns a list of two bounding boxes and we wouldn't even need the file. I don't like this Table.read code now appearing in three separate test files.

Reply:
I've added a helper function to `lsst.meas.algorithms.testUtils` to generate the common list of badPixels, and have removed `policy` (which only contained that file).